### PR TITLE
[STORM-3007] storm-mqtt-examples: fixed all checkstyle warnings

### PR DESCRIPTION
**Pull request by krichter722**

### CODE REVIEW

File: examples/storm-mqtt-examples/src/main/java/org/apache/storm/mqtt/examples/CustomMessageMapper.java [hunk 1]
```diff
@@ -15,35 +15,77 @@
...
+ * emits a tuple containing
+ * {@code user(String), deviceId(String), location(String), temperature(float),
+ * humidity(float)}.
  */
-public class CustomMessageMapper implements MqttMessageMapper {
-    private static final Logger LOG = LoggerFactory.getLogger(CustomMessageMapper.class);
-
+public final class CustomMessageMapper implements MqttMessageMapper {
+    private static final long serialVersionUID = 1L;
+    /**
```
Comment (HeartSaVioR) [inline examples/storm-mqtt-examples/src/main/java/org/apache/storm/mqtt/examples/CustomMessageMapper.java]: I guess checkstyle doesn't require fields to be commented, and the name of fields are self-descriptive. Better to revert them.

File: examples/storm-mqtt-examples/src/main/java/org/apache/storm/mqtt/examples/MqttBrokerPublisher.java [hunk 2]
```diff
@@ -15,35 +15,62 @@
...
-import java.util.Random;
-
-public class MqttBrokerPublisher {
-    private static final Logger LOG = LoggerFactory.getLogger(MqttBrokerPublisher.class);
-
+/**
+ * A MQTT example using a Storm topology.
+ */
+public final class MqttBrokerPublisher {
+    /**
```
Comment (HeartSaVioR) [inline examples/storm-mqtt-examples/src/main/java/org/apache/storm/mqtt/examples/MqttBrokerPublisher.java]: Same here: I guess checkstyle doesn't require fields to be commented, and the name of fields are self-descriptive. Better to revert them.

File: examples/storm-mqtt-examples/src/main/java/org/apache/storm/mqtt/examples/MqttBrokerPublisher.java [hunk 3]
```diff
@@ -15,35 +15,62 @@
...
     private static BrokerService broker;
-
+    /**
+     * The connection reference.
+     */
     private static BlockingConnection connection;
+    /**
+     * The maximum temperature.
+     */
+    private static final int TEMPERATUR_MAX = 100;
```
Comment (HeartSaVioR) [inline examples/storm-mqtt-examples/src/main/java/org/apache/storm/mqtt/examples/MqttBrokerPublisher.java]: TEMPERATUR_MAX -> TEMPERATURE_MAX

File: examples/storm-mqtt-examples/src/main/java/org/apache/storm/mqtt/examples/CustomMessageMapper.java (hunk 1 above)
Comment (krichter722) [reply examples/storm-mqtt-examples/src/main/java/org/apache/storm/mqtt/examples/CustomMessageMapper.java]: Correct. I assumed that `checkstyle:check` produces the same output as the checkstyle configuration run during `mvn validate`, but the former is much stricter.

**Comment by krichter722** [comment]
Requested changes incorporated. Thanks for the review.

**Pull request merged**